src/lib/project.js

```diff
@@ -1,57 +1,159 @@
-const chalk = require('chalk');
 const fs = require('fs-extra');
 const path = require('path');
 const ora = require('ora');
 const sh = require('shelljs');
 const util = require('util');
 const gittar = require('gittar');
+const { prompt } = require('enquirer');
+const { spawnSync } = require('child_process');
+const { red, green, reset } = require('chalk');
 
-const _green = chalk.green;
-const _red = chalk.red;
 const shExec = util.promisify(sh.exec);
 
 /**
  * Create a new zkApp project with recommended dir structure, Prettier config,
  * testing lib, etc. Warns if already exists and does NOT overwrite.
  * @param {string} name  Desired dir name or path. Will recursively create
  *                       dirs without overwriting existing content, if needed.
- * @return {Promise<void>}
+ * @return {promise<void>}
  */
-async function project(name) {
-  const lang = 'ts';
+async function project({ name, ui }) {
```

Comment:
Doc comments are out of date, do you mind updating them with the new arguments? :D

Reply:
thanks. will do

Reply:
done